streams/src/main/java/org/apache/kafka/streams/processor/internals/GlobalStateUpdateTask.java

```diff
@@ -114,8 +118,16 @@ public void flushState() {
         stateMgr.checkpoint(offsets);
     }
 
-    public void close() throws IOException {
+    public void close(final boolean wipeStateStore) throws IOException {
         stateMgr.close();
+        if (wipeStateStore) {
+            try {
+                log.error("Wiping state stores for global task.");
```

Comment:
This doesn't seem to be an error. Maybe info would be better? Also, I think "wipe state stores" might be confusing for a user looking at the log messages with no context. "Deleting the task directory" seems to be a more context-free statement of what we're doing.

```suggestion
                log.info("Deleting global task directory after detecting corruption.");
```